cn.navclub.vdbfx.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id" : "cn.navclub.dbfx",
```

Comment:
there don't seem to be a website responding a `navclub.cn` so it's hard to verify here that the app-id is proper.

Reply:
The domain name "navclub. CN" belongs to me. Because the ECS expired a few days ago and has not been renewed in time, I can't access it at present